release/src/main/scripts/build_release_candidate.sh

```diff
@@ -377,9 +377,7 @@ if [[ $confirmation = "y" ]]; then
   cd ${BEAM_ROOT_DIR}
   RELEASE_COMMIT=$(git rev-list -n 1 "tags/${RC_TAG}")
   # TODO(https://github.com/apache/beam/issues/20209): Don't hardcode py version in this file.
-  # TODO(https://github.com/apache/beam/issues/25649): Remove intermediate gen_protos step.
-  cd sdks/python && pip install -r build-requirements.txt && python gen_protos.py && tox -e py38-docs
-  GENERATED_PYDOC=~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_PYTHON_DOC}/${BEAM_ROOT_DIR}/sdks/python/target/docs/_build
+  cd sdks/python && pip install -r build-requirements.txt && tox -e py38-docs  GENERATED_PYDOC=~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_PYTHON_DOC}/${BEAM_ROOT_DIR}/sdks/python/target/docs/_build
```

Comment:
Looks like the spacing got messed up here, GENERATED_PYDOC should still be its own line